generators/client/templates/angular/src/main/webapp/app/entities/user/user.service.ts.ejs

```diff
@@ -38,4 +39,21 @@ export class UserService {
     const options = createRequestOption(req);
     return this.http.get<IUser[]>(this.resourceUrl, { params: options, observe: 'response' });
   }
+
+  addUserToCollectionIfMissing(userCollection: IUser[], ...usersToCheck: (IUser | null)[]): IUser[] {
+    const users: IUser[] = usersToCheck.filter(isPresent);
```

Comment:
Can we simplify parameter `usersToCheck` and can we use assignment without `isPresent`?